ci/dev.yml

```diff
@@ -36,3 +36,6 @@ dependencies:
   - pytest-cov
   - pytest-lazy-fixture
   - pytest-sugar
+  - pip
+  - pip:
+    - pytest-lazy-fixture
```

Comment:
```suggestion
```

We already have pytest-lazy-fixture in the yaml (see above from conda). Do you need to latest version from pip? Is the version not in sync with conda?